feature/bgp/policybase/otg_tests/route_installation_test/route_installation_test.go

```diff
@@ -0,0 +1,683 @@
+	policytel := statePath.PeerGroup(peerGrpName).ApplyPolicy().ImportPolicy().Get(t)
+	for _, val := range policytel {
+		if val != policy {
+			t.Errorf("Apply policy mismatch: got %v, want %v", policytel, policy)
+		}
+	}
+}
+
+// configureOTG configures the interfaces and BGP protocols on an OTG, including advertising some
+// (faked) networks over BGP.
+func configureOTG(t *testing.T, otg *ondatra.OTG, expectedRoutes int32) (gosnappi.Config, otgutils.ExpectedState) {
+
+	config := otg.NewConfig(t)
+	srcPort := config.Ports().Add().SetName("port1")
+	dstPort := config.Ports().Add().SetName("port2")
+
+	srcDev := config.Devices().Add().SetName(ateSrc.Name)
+	srcEth := srcDev.Ethernets().Add().
+		SetName(ateSrc.Name + ".eth").
+		SetPortName(srcPort.Name()).
+		SetMac(ateSrc.MAC)
+	srcIpv4 := srcEth.Ipv4Addresses().Add().
+		SetName(ateSrc.Name + ".ipv4").
+		SetAddress(ateSrc.IPv4).
+		SetGateway(dutSrc.IPv4).
+		SetPrefix(int32(ateSrc.IPv4Len))
+	srcIpv6 := srcEth.Ipv6Addresses().Add().
+		SetName(ateSrc.Name + ".ipv6").
+		SetAddress(ateSrc.IPv6).
+		SetGateway(dutSrc.IPv6).
+		SetPrefix(int32(ateSrc.IPv6Len))
+
+	dstDev := config.Devices().Add().SetName(ateDst.Name)
+	dstEth := dstDev.Ethernets().Add().
+		SetName(ateDst.Name + ".eth").
+		SetPortName(dstPort.Name()).
+		SetMac(ateDst.MAC)
+	dstIpv4 := dstEth.Ipv4Addresses().Add().
+		SetName(ateDst.Name + ".ipv4").
+		SetAddress(ateDst.IPv4).
+		SetGateway(dutDst.IPv4).
+		SetPrefix(int32(ateDst.IPv4Len))
+	dstIpv6 := dstEth.Ipv6Addresses().Add().
+		SetName(ateDst.Name + ".ipv6").
+		SetAddress(ateDst.IPv6).
+		SetGateway(dutDst.IPv6).
+		SetPrefix(int32(ateDst.IPv6Len))
+
+	srcBgp4Name := ateSrc.Name + ".bgp4.peer"
+	srcBgp6Name := ateSrc.Name + ".bgp6.peer"
+	srcBgp := srcDev.Bgp().
+		SetRouterId(srcIpv4.Address())
+	srcBgp4Peer := srcBgp.Ipv4Interfaces().Add().
+		SetIpv4Name(srcIpv4.Name()).
+		Peers().Add().
+		SetName(srcBgp4Name).
+		SetPeerAddress(srcIpv4.Gateway()).
+		SetAsNumber(ateAS).
+		SetAsType(gosnappi.BgpV4PeerAsType.EBGP)
+	srcBgp6Peer := srcBgp.Ipv6Interfaces().Add().
+		SetIpv6Name(srcIpv6.Name()).
+		Peers().Add().
+		SetName(srcBgp6Name).
+		SetPeerAddress(srcIpv6.Gateway()).
+		SetAsNumber(ateAS).
+		SetAsType(gosnappi.BgpV6PeerAsType.EBGP)
+
+	dstBgp4Name := ateDst.Name + ".bgp4.peer"
+	dstBgp6Name := ateDst.Name + ".bgp6.peer"
+	dstBgp := dstDev.Bgp().
+		SetRouterId(dstIpv4.Address())
+	dstBgp4Peer := dstBgp.Ipv4Interfaces().Add().
+		SetIpv4Name(dstIpv4.Name()).
+		Peers().Add().
+		SetName(dstBgp4Name).
+		SetPeerAddress(dstIpv4.Gateway()).
+		SetAsNumber(ateAS).
+		SetAsType(gosnappi.BgpV4PeerAsType.EBGP)
+	dstBgp6Peer := dstBgp.Ipv6Interfaces().Add().
+		SetIpv6Name(dstIpv6.Name()).
+		Peers().Add().
+		SetName(dstBgp6Name).
+		SetPeerAddress(dstIpv6.Gateway()).
+		SetAsNumber(ateAS).
+		SetAsType(gosnappi.BgpV6PeerAsType.EBGP)
+
+	prefixInt4, _ := strconv.Atoi(strings.Split(advertisedRoutesv4CIDR, "/")[1])
```

Comment:
Define separate constants for the prefixlen and the IP netblock? We want to avoid reprocessing of test parameters because that's error prone.

Reply:
Done